pyproject.toml

```diff
@@ -0,0 +1,156 @@
+# https://peps.python.org/pep-0517/
+[build-system]
+requires = ["setuptools>=45", "wheel"]
+build-backend = "setuptools.build_meta"
+
+# https://peps.python.org/pep-0621/
+[project]
+name = "napari-aicsimageio"
+# Do not edit this string manually, always use bumpversion
+# Details in CONTRIBUTING.rst
+version = "0.7.0"
+description = "AICSImageIO bindings for napari"
+keywords = [
+  "aicsimageio",
+  "CZI",
+  "image reading",
+  "imageio",
+  "LIF",
+  "metadata",
+  "napari",
+  "ND2",
+  "TIFF",
+]
+readme = "README.md"
+requires-python = ">=3.8"
+license = { text = "GPLv3" }
+authors = [
+  { email = "evamaxfieldbrown@gmail.com", name = "Eva Maxfield Brown" },
+  { email = "talley.lambert@gmail.com", name = "Talley Lambert" },
+]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Science/Research",
+  "Framework :: napari",
+  "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Topic :: Scientific/Engineering :: Bio-Informatics",
+  "Topic :: Scientific/Engineering :: Image Processing",
+  "Topic :: Scientific/Engineering :: Information Analysis",
+  "Topic :: Scientific/Engineering :: Visualization",
+  "Topic :: Scientific/Engineering",
+]
+dynamic = []
+dependencies = [
+  "aicsimageio[all]>=4.6.3",
+  "fsspec[http]",            # no version pin, we pull from aicsimageio
+  "napari>=0.4.11",
+  "psutil>=5.7.0",
+  # Formats not included by default with aicsimageio
+  "aicspylibczi>=3.0.5",
+  "bioformats_jar",
+  "readlif>=0.6.4",
+]
+
+[project.urls]
+Homepage = "https://github.com/AllenCellModeling/napari-aicsimageio"
+"Bug Tracker" = "https://github.com/AllenCellModeling/napari-aicsimageio/issues"
+Documentation = "https://github.com/AllenCellModeling/napari-aicsimageio#README.md"
+"User Support" = "https://github.com/AllenCellModeling/napari-aicsimageio/issues"
+
+# extras
+# https://peps.python.org/pep-0621/#dependencies-optional-dependencies
+[project.optional-dependencies]
+test = [
```

Comment:
these are your extras.  Note that it's kinda best just to run `pip install -e .[test,dev]` if you want multiple extras (rather than duplicating them in multiple extras)